src/main/java/com/xebialabs/overthere/cifs/winrm/KerberosJaasConfiguration.java

```diff
@@ -49,20 +49,26 @@
             options.put("debug", "true");
         }
         
-        if (ticketCache) {
-        	options.put("useTicketCache", "true");
-        } else {
-        	options.put("useTicketCache", "false");
-        }
-        
         options.put("refreshKrb5Config", "true");
         
         if (JavaVendor.isIBM()) {
             options.put("credsType", "initiator");
+            
+            if (ticketCache) {
+            	options.put("useDefaultCcache", "true");
+            } else {
+            	options.put("useDefaultCcache", "false");
```

Comment:
Did you mean to write two c's there? According to [this document](https://www-304.ibm.com/support/knowledgecenter/#!/SS69YH_6.0.0/com.ibm.spss.cads.config.doc/model_management/thin/server_process_credential.html) the property is called `useDefaultCache`, with a single c instead of Cc.